packages/contracts/contracts/L1/interfaces/IAssetManager.sol

```diff
@@ -460,6 +468,14 @@ interface IAssetManager {
      */
     function deposit(uint128 assets) external;
 
+    /**
+     * @notice Withdraw KRO. To withdraw KRO, the validator should be initiated and MIN_DELEGATION_PERIOD
+     *         should be passed after the last deposit time.
+     *
+     * @param assets The amount of KRO to withdraw.
+     */
+    function withdraw(uint128 assets) external;
```

Comment:
At first, I thought that the caller of `withdraw` function is withdrawAccount, not validator. 
But I think this is also possible. We need to discuss about this regarding operator experience or security wise.

Reply:
I think I misunderstood the intention. I'll fix the interface and the implementation as you said.